environment.yml

```diff
@@ -1,13 +1,18 @@
 name: cookiecutter-birdhouse
 channels:
-- conda-forge
+#- conda-forge
```

Comment:
I suggest we remove the comments to keep it clean.

Reply:
done.